<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="msgBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="msgTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="usrTblWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>261</width>
     <height>381</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP地址</string>
    </property>
   </column>
  </widget>
  <widget class="QFontComboBox" name="fontCbx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeCbx">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="boldTBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>加粗</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="italicTBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>倾斜</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="underlineTBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="colorTBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>更改颜色字体</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="sendTBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>传输文件</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="saveTBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>保存聊天记录</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="clearTBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>清空聊天记录</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>711</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="sendBtn">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>156</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="usrNumLbl">
      <property name="text">
       <string>在线人数</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>156</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="exitBtn">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Widget</sender>
   <signal>destroyed(QObject*)</signal>
   <receiver>exitBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
